### ISSUE TITLE
test throws warning

### ISSUE THREAD

**Issue by berndbischl** [berndbischl/ParamHelpers#112]
Warnings -----------------------------------------------------------------------
1. requires chains work (@test_generateDesign.R#191) - generateDesign could only produce 3 points instead of 4!

this should be handled, tests should not do this.

**Comment by jakob-r** [comment]
What is strange is, that it woks for me in interactive
```r
> ps8 = makeParamSet(
+ makeLogicalLearnerParam("a", default = FALSE),
+ makeLogicalLearnerParam("b", default = FALSE, requires = quote(a == TRUE)),
+ # FIXME: shouldn't need to make the chain explicit
+ makeLogicalLearnerParam("c", default = FALSE, requires = quote(b == TRUE && a == TRUE))
+ )
> des = generateDesign(4L, par.set = ps8)
> des
       a     b     c
1  FALSE    NA    NA
3   TRUE FALSE    NA
4   TRUE  TRUE FALSE
41  TRUE  TRUE  TRUE
```

**Comment by jakob-r** [comment]
Might be related to #95

**Comment by berndbischl** [comment]
was fixed in #118
